### PR TITLE
Swap tailwind for new design; hardcode old tailwind.css

### PR DESCRIPTION
**Pull request by martinemde**
This hardcodes the old generated tailwind.css with `tw-` prefix. This hardcoded css will remain until the pages are replaced with the new design. We now compute the new design's tailwind only, though it's not used anywhere except local development, so far.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,7 +15,6 @@
     <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
 
     <%= stylesheet_link_tag("application") %>
-    <%= stylesheet_link_tag("tailwind", "data-turbo-track": "reload") %>
```
Comment (segiddins) [inline app/views/layouts/application.html.erb:18]: shouldn't this now link the deprecated generated file?

Comment (martinemde) [reply app/views/layouts/application.html.erb:18]: It's compiled into the application.css now.

Comment (martinemde) [reply app/views/layouts/application.html.erb:18]: I did check the views that rely on `tw-` classes and they look the same as before, e.g. trusted publishing.

File: app/assets/stylesheets/application.tailwind.css [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  @font-face {
+    font-family: "Titillium Web";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(/fonts/Roboto.woff2) format('woff2');
```
Comment (martinemde) [inline app/assets/stylesheets/application.tailwind.css:12]: Yes the font here is wrong but it doesn't affect rendering. I'll update it when we actually merge the design. (I couldn't find the best way to do this without a src).

File: config/tailwind.config.js [hunk 3]
```diff
@@ -15,17 +14,105 @@ module.exports = {
...
-          "Helvetica Neue",
-          "Helvetica",
-          "Arial",
-          ...defaultTheme.fontFamily.sans,
-        ],
+        sans: ['"Titillium Web"', ...defaultTheme.fontFamily.sans],
+        mono: ['"Fira Code"', ...defaultTheme.fontFamily.mono],
+      },
+      fontSize: {
+        lg:   ['1.4375rem', '2.156rem'], /* Base/01 23px, 34.5px */
```
Comment (colby-swandale) [inline config/tailwind.config.js:21]: Can I ask where these sizes are? Adjusting font size would also mean we need to adjust line height and other typography related values.

Comment (martinemde) [reply config/tailwind.config.js:21]: Copied from the figma foundation design. Line heights are correct so far as I've implemented the design.

Comment (martinemde) [reply config/tailwind.config.js:21]: The second number of each size is the line height from the design.

**Comment by martinemde** [comment]
Any blockers on this merging? This is the smallest first piece I can manage, just implementing the foundational colors and font sizes/line heights from the [figma design](https://www.figma.com/design/glT62VyAlJak6eh9d0KRSb/%F0%9F%92%8E-Ruby?node-id=102-991&m=dev).

**Review by colby-swandale — APPROVED** [review]

**Pull request merged**